### DISCUSSION TITLE
Correlation matrix

### DISCUSSION THREAD

**Discussion by leewalton2014** [opensource9ja/danfojs#371] [Q&A] [answered]
I have my dataframe with all of my data how do I apply the corr() method to this I tried df.corr() but console reported an error and the documentaation I am viewing has limited guidelines for use.

**Comment by risenW** [comment, marked as answer]
> I have my dataframe with all of my data how do I apply the corr() method to this I tried df.corr() but console reported an error and the documentaation I am viewing has limited guidelines for use.

This feature is not support yet
